### DISCUSSION TITLE
How to upgrade to 4.6.0 on Raspberry 4B/Ubuntu Server 22.04 LTS

### DISCUSSION THREAD

**Discussion by tqchiem** [Motion-Project/motion#1734] [General]
I am new to "motion" and have managed to display video streams from 4 WiFi cameras on my own web page running on Raspberry 4B with Ubuntu Server 22.04 LTS 64-bit. I installed motion using "sudo apt-get install motion". This installed version 4.3.2-1. It says this version is the newest version when I tried to update it. How do I update it to version 4.6.0 without having to compile the source (reasons - I am not a software person and want to test motion in its known-good package)?.

Unrelated to this upgrade, during testing I find:
1). RPi4 takes a much longer time (minutes) to boot with motion
2). After "sudo systemctl restart motion", the %CPU jumps from about 12% to over 170% and then fluctuates between 98-115% (from "top" command) with %MEM of 8. Note that there is no motion detection configured, just rtsp from the 4 cameras.

Any help/comments would be very much appreciated. Thanks.

**Comment by hpojak** [comment]
A late comment...
You can download a package according to https://motion-project.github.io/motion_build.html
See the paragraph "Installing with a release deb package".
